plugins/linear-genome-view/src/LinearGenomeView/components/OverviewScaleBar.tsx

```diff
@@ -306,6 +306,7 @@ function OverviewScaleBar({
   const overview = Base1DView.create({
     displayedRegions: JSON.parse(JSON.stringify(displayedRegions)),
     interRegionPaddingWidth: 0,
+    minimumBlockWidth: model.minimumBlockWidth,
   })
   overview.setVolatileWidth(width)
   overview.showAllRegions()
```

Comment:
Whoa, why are we creating MST models inside react components in the first place? Must have missed this in the PR that brought it in originally. This seems wrong, but not related to this PR here.

cc @teresam856

Reply:
What would be a good alternative? It is used to take the displayedRegions of the current LGV, and use it to power the overview bar

Reply:
@rbuels can you clarify what you meant of if this needs changes ?

Reply:
Line 306 is instantiating an MST model. And the code lies inside a react component.

This seems like a bad place to have this.

Reply:
This PR did not introduce this though.

Reply:
>This seems like a bad place to have this.

The MST model here is used only for computation, not for observability. What is bad about this?

Reply:
i guess nothing